### DISCUSSION TITLE
Version upgrade from vue 2 to vue 3

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#13051] [CLI - PWA mode] [answered]
Hello Friends 

I have a vue 2 app together with quasar on top which run on SSR mode. i want to upgade to vue 3 with the lastest version of quasar, but i dont have any idea how i will processed. i have gone through upgrade guide but i got lost my application stops working.
Can someone guide me step by step what should i do please

Thanks

**Comment by MilosPaunovic** [comment, marked as answer]
The official guide to upgrading from Quasar 1 (with Vue 2) to Quasar 2 (with Vue 3) is very comprehensive and, if you follow it step by step, you should achieve transition smootly.

[https://quasar.dev/start/upgrade-guide#migrate-to-v2-from-v1](https://quasar.dev/start/upgrade-guide#migrate-to-v2-from-v1)

**Comment by altShiftDev** [comment]
The guide is gone now but for others looking for this in the future use this mirror from the waybackmachine: https://web.archive.org/web/20210922215516/https://quasar.dev/start/upgrade-a
